lib/datadog/opentelemetry/metrics.rb

```diff
@@ -105,6 +100,15 @@ def configure_otlp_exporter(provider)
       rescue LoadError => e
         @logger.warn("Could not load OTLP metrics exporter:  #{e.class}: #{e}")
       end
+
+      # Returns metrics config value if explicitly set, otherwise falls back to exporter config or computed default value.
+      def get_metrics_config_with_fallback(metrics_config, exporter_config, option_name, computed_default = nil)
+        if metrics_config.using_default?(option_name)
+          exporter_config.send(option_name) || computed_default
```

Comment:
it's a good practice to use `public_send` instead of `send`, unless you want to call private methods